### ISSUE TITLE
Wrong URL for checking the backend connection status

### ISSUE THREAD

**Issue by azatsarynnyy** [theia-ide/theia#2196]
I've built and run Theia app from the following package.json:
```json
{
  "private": true,
  "name": "browser-app",
  "version": "0.0.0",
  "dependencies": {
    "@theia/core": "latest",
    "@theia/filesystem": "latest",
    "@theia/workspace": "latest",
    "@theia/preferences": "latest",
    "@theia/navigator": "latest",
    "@theia/process": "latest",
    "@theia/terminal": "latest",
    "@theia/editor": "latest",
    "@theia/languages": "latest",
    "@theia/markers": "latest",
    "@theia/monaco": "latest",
    "@theia/typescript": "latest",
    "@theia/messages": "latest"
  },
  "devDependencies": {
    "@theia/cli": "latest"
  },
  "scripts": {
    "prepare": "theia build --mode development",
    "start": "theia start",
    "watch": "theia build --watch --mode development"
  },
  "theia": {
    "target": "browser"
  }
}
```

The status bar color signals that the Theia's backend is offline but actually it's reachable.
The browser's DevTools shows that frontend tries to ping the connection status service on a wrong URL: `http://localhost:3000//alive`.

**Comment by AlexTugarev** [comment]
This was fixed in #2074, but not yet released.

Can you retry with `next` instead of `latest`?

**Comment by azatsarynnyy** [comment]
Yes, it works with the `next` version.
Thank you!
